### DISCUSSION TITLE
English voice in DAOTools

### DISCUSSION THREAD

**Discussion by Doreker** [humhue/DAOTools#14] [Q&A] [answered]
Hello, I installed your multilingual mod for QUDAO, the text returned to the originally installed one, but some of the dialogues with Allister and Morrigan sound in English instead of the installed one. Is there a way to fix this or are you no longer doing this?

**Comment by Doreker** [comment]
Remove all .fsb and .fev files from the the sound directory in [...]\qwinn_fixpack_3\core\audio\sound but it didn't help

**Reply by humhue** [reply, marked as answer]
That should have worked. Are you sure you extracted the files from the dazip, deleted all fsv and fev files there, zipped the folder again, then deleted QUDAO from the mod list and added it again?
